components/CartProductListItem.vue

```diff
@@ -55,6 +61,12 @@ export default {
     productURL(url) {
       return this.combineURLs(`${this.$store.getters.apiURL}`, url)
     },
+    addItem(item) {
+      return this.$store.dispatch("cart/addItem", item)
+    },
+    decreaseItem(item) {
+      return this.$store.dispatch("cart/decreaseItem", item)
+    },
```

Comment:
Shouldn't the `mapActions` above export it? I believe this is redundant

Reply:
I see your change, but I meant to delete the selected lines and add `addItem` and `decreaseItem` which you have just removed to `mapActions`, because it aliases the functions to work exactly in the way your newly created functions work